pkg/image/apis/image/register.go

```diff
@@ -54,6 +56,9 @@ func addLegacyKnownTypes(scheme *runtime.Scheme) error {
 		&ImageStreamTagList{},
 		&ImageStreamImage{},
 		&ImageStreamImport{},
+
+		// exposed for backwards compatibility
+		&docker.DockerImage{},
```

Comment:
Move this into the register for the /docker package?

Reply:
And then call the register from this method?  We need it in the v1 scheme, so do we want the /docker package adding things to a v1 group?